site/_posts/2017-09-04-the-school-year-begins.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Last Week in ManageIQ: The school year begins - let's improve our GPA"
+author: skateman
+date: 2017-09-05
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello, [Dávid](https://github.com/skateman) here from the UI team. Let's have a quick recap about what happened in the last week on the magical land of ManageIQ.
+
+Honestly, I gave up on counting how many repositories we are currently maintaining. The only number I come up with is ~220 and this is the number of PRs we merged last week. Around 50% of these PRs got merged into [core][manageiq PRs merged] and into the [two][manageiq-ui-classic PRs merged] [UI][[manageiq-ui-service PRs merged]] repos, feel free to check them out.
```

Comment:
Think there are too many `[` and `]` in that last link which is causing it to fail to render a link!